cli/pkg/cmd/create/create.go

```diff
@@ -88,69 +84,9 @@ func create(cmd *cobra.Command, args []string) {
 	fmt.Fprintln(w)
 	fmt.Fprintln(w, providerName, "provider output:")
 	fmt.Fprintln(w)
-	context, err := provider.Provision(providerConfig, workspace.ConfigDir{Path: installDir})
+	err = provider.Provision(providerConfig, workspace.ConfigDir{Path: installDir})
 	fmt.Fprintln(w)
 	if err != nil {
 		console.Exit("could not install Airy: ", err)
 	}
-	if providerName == "minikube" { // TEMP fix to keep minikube working
-		clientset, err := context.GetClientSet()
-		if err != nil {
-			console.Exit("could not get clientset: ", err)
-		}
-
-		if err = context.Store(); err != nil {
-			console.Exit("could not store the kube context: ", err)
-		}
-
-		helm := helm.New(clientset, version, namespace, dir.GetAiryYaml())
-		if err := helm.Setup(); err != nil {
-			console.Exit("setting up Helm failed with err: ", err)
-		}
-
-		fmt.Println("🚀 Starting core with default components")
-
-		if err := helm.InstallCharts(); err != nil {
-			console.Exit("installing Helm charts failed with err: ", err)
-		}
-
-		if err = provider.PostInstallation(providerConfig, namespace, dir); err != nil {
-			console.Exit("failed to run post installation hook: ", err)
-		}
-
-		fmt.Println("🎉 Your Airy Core is ready")
-
-		coreConfig, err := k8s.GetCmData("core-config", namespace, clientset)
-		if err != nil {
-			console.Exit("failed to get hosts from installation")
-		}
-
-		fmt.Println("\t 👩‍🍳 Available hosts:")
-		for hostName, host := range coreConfig {
-			switch hostName {
-			case "HOST":
-				fmt.Printf("\t\t %s:\t %s", "Host", host)
-				fmt.Println()
-			case "API_HOST":
-				fmt.Printf("\t\t %s:\t %s", "API", host)
-				fmt.Println()
-			case "NGROK":
-				fmt.Printf("\t\t %s:\t %s", "NGROK", host)
-				fmt.Println()
-			}
-		}
-
```

Comment:
I guess we still want to keep everything below this line, right? (printing docs, storing the provider in the config, tracking)

Reply:
All the output is printed currently by Terraform. When Terraform finishes, it prints out the outputs which have the API/UI url and the link for the docs. An alternative would be to mute the Terraform output and once Terraform finishes successfully, to get the outputs and print them from the CLI.

Reply:
Understood, but what about the tracking and storing the provider in the `cli.yaml`?

Reply:
The tracking was moved to the Provision stage. But I guess it makes sense that is also in the  PostInstallation, so that we record an event when the installation finished.

Good idea to store the provider in the `cli.yaml`. I was thinking perhaps in the `install.flags`. Will have a look :+1:.